app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/AgendaItemDetailFragment.kt

```diff
@@ -240,10 +265,76 @@ class AgendaItemDetailFragment : Fragment(R.layout.fragment_agenda_item_detail)
             descriptionTextView.isClickable = isEditing
 
             time.isClickable = isEditing
+            timeButton.isVisible = isEditing
+            timeButton.isClickable=isEditing
             date.isClickable = isEditing
+            dateButton.isVisible = isEditing
+            dateButton.isClickable = isEditing
```

Comment:
I think I mentioned this the last time, but this should be `dateButton.isEnabled = isEditing`